binding.gyp

```diff
@@ -38,6 +38,23 @@
 					} ]
 				},
 				{
+					"target_name": "copyapis",
+					"type": "none",
+					"conditions": [ [ "'<!(echo $BUILD_NPM_PACKAGE)'=='y'", {
+						"actions": [ {
+							"action_name": "copyapis",
+							"message": "Copying JS APIs",
+							"outputs": [ "./uart.js" ],
```

Comment:
In fact, the hardware bindings will fail exactly as intended, in that the promises they return will be rejected, because the promise function will throw an exception when it attempts to call ```soletta.sol_<hardware_api>_*()```. It's not as clean as rejecting ```if ( typeof soletta.sol_<hardware_api>_*() === "undefined" )```, but it works just as well.